avocado.spec

```diff
@@ -7,14 +7,19 @@
 Summary: Framework with tools and libraries for Automated Testing
 Name: avocado
 Version: 45.0
-Release: 1%{?dist}
+Release: 2%{?dist}
 License: GPLv2
 Group: Development/Tools
 URL: http://avocado-framework.github.io/
 Source0: https://github.com/avocado-framework/%{name}/archive/%{commit}/%{name}-%{version}-%{shortcommit}.tar.gz
 BuildArch: noarch
-Requires: python, python-requests, fabric, pyliblzma, libvirt-python, gdb, gdb-gdbserver, python-stevedore, python-yaml, procps-ng, aexpect
-BuildRequires: python2-devel, python-setuptools, python-docutils, python-mock, python-psutil, python-sphinx, python-requests, aexpect, pystache, yum, python-stevedore, python-lxml, perl-Test-Harness, fabric, python-flexmock, python-yaml, procps-ng
+Requires: python, python-requests, pyliblzma, gdb, gdb-gdbserver, python-stevedore, python-yaml, procps-ng
+BuildRequires: python2-devel, python-setuptools, python-docutils, python-mock, python-psutil, python-sphinx, python-requests, pystache, yum, python-stevedore, python-lxml, perl-Test-Harness, fabric, python-flexmock, libvirt-python, python-yaml, procps-ng
+%if 0%{?fedora}
+BuildRequires: python-aexpect
+%else
+BuildRequires: aexpect
```

Comment:
OK so if I understand this correctly the official package is called `python-aexpect`, but the upstream one is `aexpect`, right? I think we should rename the upstream spec file...